common/scala/src/main/scala/whisk/core/database/CouchDbRestStore.scala

```diff
@@ -52,7 +52,8 @@ class CouchDbRestStore[DocumentAbstraction <: DocumentSerializer](dbProtocol: St
                                                                   dbPassword: String,
                                                                   dbName: String,
                                                                   useBatching: Boolean = false,
-                                                                  val inliningConfig: InliningConfig)(
+                                                                  val inliningConfig: InliningConfig,
+                                                                  val attachmentStore: Option[AttachmentStore])(
```

Comment:
attachmentStore not in scaladoc.

Reply:
It is currently not part of API but instead used by`CouchDBStoreBehaviorBase` for test purpose. Code dealing with `ArtifactStore` is not aware of `AttachmentStore` for now